.circleci/config.yml

```diff
@@ -151,6 +151,16 @@ jobs:
       PYTEST_ARGS: -k FIREFOX_DESKTOP -m remote_settings --reruns 1
     steps:
       - checkout
+      - run:
+          name: Check file paths
+          command: |
+            if git diff --name-only main HEAD | grep -E 'experimenter/'
+              then
+                echo "Continuing"
+              else
+                echo "No targeting changes, skipping"
+                circleci-agent step halt
+            fi
```

Comment:
You can define a top level `commands` section with reusable steps that you can then call in here (with parameters if, for example, you want to pass the repo path to check for differences). https://circleci.com/docs/reusing-config/#authoring-reusable-commands